Drop after lang/rust >= 1.86 update per https://github.com/rust-lang/rust/commit/cbd44d799800

error[E0658]: cannot cast `dyn ai::Action<S, R>` to `dyn std::any::Any`, trait upcasting coercion is experimental
   --> rtsim/src/ai/mod.rs:126:16
    |
126 |         match (other as &dyn Any).downcast_ref::<Self>() {
    |                ^^^^^
    |
error[E0658]: cannot cast `dyn ai::Action<S, R>` to `dyn std::any::Any`, trait upcasting coercion is experimental
   --> rtsim/src/ai/mod.rs:375:16
    |
375 |         match (other as &dyn Any).downcast_ref::<Self>() {
    |                ^^^^^
    |
    = note: see issue #65991 <https://github.com/rust-lang/rust/issues/65991> for more information
    = help: add `#![feature(trait_upcasting)]` to the crate attributes to enable
    = note: this compiler was built on 2025-01-07; consider upgrading it if it is out of date
    = note: required when coercing `&(dyn ai::Action<S, R> + 'static)` into `&(dyn std::any::Any + 'static)`

--- rtsim/src/lib.rs.orig	2025-02-19 10:27:05 UTC
+++ rtsim/src/lib.rs
@@ -137,6 +137,7 @@
 //! See [`rule::npc_ai`].
 
 #![feature(never_type, let_chains, binary_heap_drain_sorted)]
+#![feature(trait_upcasting)]
 
 pub mod ai;
 pub mod data;
